api/src/main/java/io/strimzi/api/kafka/model/kafka/cruisecontrol/CruiseControlSpec.java

```diff
@@ -109,6 +110,16 @@ public void setConfig(Map<String, Object> config) {
         this.config = config;
     }
 
+    @Description("The Cruise Control `ApiUsers` configuration")
```

Comment:
Maybe you should explain what this is? MAybe something like this?

```suggestion
    @Description("Configuration of the Cruise Control REST API users.")
```